<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image:  url(:/background2.png);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>ImageCompressor</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>481</width>
     <height>651</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="ClickedLabel" name="originLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>421</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 2px;
border-style: solid;
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="compressedLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>421</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 2px;
border-style: solid;
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>40</y>
     <width>231</width>
     <height>651</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="openLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 1px;
border-style: solid;
border-color: black;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 1px;
border-style: solid;
border-color: black;
border-radius:10px</string>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 1px;
border-style: solid;
border-color: black;
border-radius:10px</string>
    </property>
    <property name="text">
     <string>编码</string>
    </property>
   </widget>
   <widget class="QSlider" name="errorHorizontalSlider">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>40</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Myungjo Std M</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>误差容许量</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 1px;
border-style: solid;
border-color: black;
border-radius:10px</string>
    </property>
    <property name="text">
     <string>解码</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="errorSpinBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 1px;
border-style: solid;
border-color: black;
border-radius:8px</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ClickedLabel</class>
   <extends>QLabel</extends>
   <header>clickedlabel.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
